File: Python/1010.py
peca1 = input().split()
num1 = int(peca1[1])
valor1 = float(peca1[2])

peca2 = input().split()
num2 = int(peca2[1])
valor2 = float(peca2[2])

preco = (num1*valor1) + (num2*valor2)

print(f'VALOR A PAGAR: R$ {preco:.2f}')

File: Ruby/1_Iniciante/1010.rb
peca1 = gets.split()
num1 = peca1[1].to_i
valor1 = peca1[2].to_f

peca2 = gets.split()
num2 = peca2[1].to_i
valor2 = peca2[2].to_f

preco = (num1*valor1) + (num2*valor2)

puts "VALOR A PAGAR: R$ %.2f" % preco
